src/main/resources/static/data-upload.js

```diff
@@ -109,18 +109,39 @@ function retrieveSessions() {
         }
     });
 
-    $.ajax({
-        url: "/unfinished-sessions",
-        type: "get",
-        success: handle_retrieval,
-        error: handle_error
+    //this commented code is for retrieving the sessions in real situations
+    // $.ajax({
```

Comment:
Again, you do not need to uncomment code to keep it around. But if you are more comfortable with this approach, let's keep it for now. But do keep it in mind that this is not a good way.